<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerClass</class>
 <widget class="QMainWindow" name="ServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="statusText">
       <property name="text">
        <string>&lt;Status&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="statusList"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="userList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>71</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="server.qrc"/>
 </resources>
 <connections/>
</ui>
